application/classes/Ushahidi/Core.php

```diff
@@ -863,6 +863,9 @@ public static function init()
 		 * Attach database config
 		 */
 		self::attached_db_config();
+
+		// override default-language with site-language
+		I18n::lang(Kohana::$config->load('site.language'));
```

Comment:
Should this pull from either a user's language or the site's language?

Reply:
Its the site's language